code/modules/paperwork/paper.dm

```diff
@@ -454,6 +454,23 @@
 	ui_interact(user)
 	return ..()
 
+/// Secondary right click interaction to quickly stamp things
+/obj/item/paper/item_interaction_secondary(mob/living/user, obj/item/tool, list/modifiers)
+	var/list/writing_stats = tool.get_writing_implement_details()
+
+	if(!length(writing_stats))
+		return NONE
+	if(writing_stats["interaction_mode"] != MODE_STAMPING)
+		return NONE
+	if(!user.can_read(src) || user.is_blind()) // Just leftclick instead
+		return NONE
+
+	add_stamp(writing_stats["stamp_class"], rand(1, 300), rand(1, 400), stamp_icon_state = writing_stats["stamp_icon_state"])
+	user.visible_message(span_notice("[user] quickly stamps [src] with \the [tool]"))
+	balloon_alert(user, "paper stamped")
+	playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, vary = TRUE)
```

Comment:
this balloon alert isn't necessary, given we will have audio feedback (the sound effect), visual feedback (the paper sprite changing) and a visible message

Reply:
```suggestion
	playsound(src, 'sound/items/handling/standard_stamp.ogg', 50, vary = TRUE)
```
ok